test/build/core-flags/core-flags.test.js

```diff
@@ -239,15 +239,17 @@ describe("core flags", () => {
       expect(stdout).toContain(`devtool: 'source-map'`);
     });
 
-    it("should allow string value devtool option using alias", async () => {
+    // TODO: Enable alias with webpack 5
+    it.skip("should allow string value devtool option using alias", async () => {
```

Comment:
Why skip?

Reply:
We don't have alias property coming from webpack.

https://github.com/webpack/webpack/blob/8f87b50dc7ac24eb5c91fd0d55a22e34e252863c/test/__snapshots__/Cli.basictest.js.snap#L416

We had defined alias for only few config options which we had for webpack 4 which we removed in this PR.